docs/ref/primitives.rst

```diff
@@ -68,6 +85,26 @@ These are the lowest level building blocks for creating parsers.
       >>> string_from("y", "yes").parse("yes")
       'yes'
 
+
+.. function:: match_item(item, description=None)
+
+   Returns a parser that tests the next item (or character) from the stream (or
+   string) for equality against the provided item. Optionally a string
+   description can be passed.
+
+   Parsing a string:
+
+   >>> letter_A = match_item('A')
+   >>> letter_A.parse_partial("ABC")
+   ('A', 'BC')
+
+   Parsing a list of tokens:
+
+   >>> class SPACE: pass
+   >>> hello = match_item('hello')
+   >>> hello.parse_partial(["hello", "how", "are", "you"])
```

Comment:
Let's pick either single or double quotes and use them consistently. I'm fine with either.

> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. (PEP 8)